features/code_generation/developer_generates_return_constant.feature

```diff
@@ -314,3 +314,195 @@ Feature: Developer generates a method returning a constant
+
+      class MyFeature
+      {
+          public function isActive()
+          {}
+      }
+
+      """
+    When I run phpspec with the option "fake" and answer "y" when asked if I want to generate the code
+    Then the class in "src/CodeGeneration/ConstantExample9/MyFeature.php" should contain:
+      """
+      <?php
+
+      namespace CodeGeneration\ConstantExample9;
+
+      class MyFeature
+      {
+          public function isActive()
+          {
+              return false;
+          }
+      }
+
+      """
+
+  Scenario: Generating a scalar return type for has positive matcher when method exists
+    Given the spec file "spec/CodeGeneration/ConstantExample10/MyFeatureSpec.php" contains:
+      """
+      <?php
+
+      namespace spec\CodeGeneration\ConstantExample10;
+
+      use PhpSpec\ObjectBehavior;
+
+      class MyFeatureSpec extends ObjectBehavior
+      {
+          function it_should_have_availability()
+          {
+              $this->shouldHaveAvailability();
+          }
+      }
+
+      """
+    And the class file "src/CodeGeneration/ConstantExample10/MyFeature.php" contains:
+      """
+      <?php
+
+      namespace CodeGeneration\ConstantExample10;
+
+      class MyFeature
+      {
+          public function hasAvailability()
+          {}
+      }
+
+      """
+    When I run phpspec with the option "fake" and answer "y" when asked if I want to generate the code
+    Then the class in "src/CodeGeneration/ConstantExample10/MyFeature.php" should contain:
+      """
+      <?php
+
+      namespace CodeGeneration\ConstantExample10;
+
+      class MyFeature
+      {
+          public function hasAvailability()
+          {
+              return true;
+          }
+      }
+
+      """
+
+  Scenario: Generating a scalar return type for has negative matcher when method exists
+    Given the spec file "spec/CodeGeneration/ConstantExample11/MyFeatureSpec.php" contains:
+      """
+      <?php
+
+      namespace spec\CodeGeneration\ConstantExample11;
+
+      use PhpSpec\ObjectBehavior;
+
+      class MyFeatureSpec extends ObjectBehavior
+      {
+          function it_should_not_have_availability()
+          {
+              $this->shouldNotHaveAvailability();
+          }
+      }
+
+      """
+    And the class file "src/CodeGeneration/ConstantExample11/MyFeature.php" contains:
+      """
+      <?php
+
+      namespace CodeGeneration\ConstantExample11;
+
+      class MyFeature
+      {
+          public function hasAvailability()
+          {}
+      }
+
+      """
+    When I run phpspec with the option "fake" and answer "y" when asked if I want to generate the code
+    Then the class in "src/CodeGeneration/ConstantExample11/MyFeature.php" should contain:
+      """
+      <?php
+
+      namespace CodeGeneration\ConstantExample11;
+
+      class MyFeature
+      {
+          public function hasAvailability()
+          {
+              return false;
+          }
+      }
+
+      """
```

Comment:
Please add a line break

Reply:
Added in 9e4a84afd4038b8911fe216409ccdd92f99ef557